src/mappers.ts

```diff
@@ -45,16 +47,15 @@ export function toBook(obj: any): Book {
   if (!obj.title) {
     throw new Error("Missing property 'title' in book");
   }
-
-  // if (!obj.author) {
-  //   throw new Error("Missing property 'author' in book");
-  // }
-  // if (!obj.isbn) {
-  //   throw new Error("Missing property 'isbn' in book");
-  // }
-  // if (!obj.summary) {
-  //   throw new Error("Missing property 'summary' in book");
-  // }
+  if (!obj.author) {
```

Comment:
title and author can be empty strings. Will those trigger errors here? If so, should this be fixed when the book is inserted (reject book insertion if it doesn't have author or title)?

Reply:
Trying to set a title or author to an empty string in the add new book modal will prevent submitting the form. So there should be no way for the user to add a book that doesn't have a title or author. Though we probably shouldn't rely on that completely. I'll change these too.